pkg/handlers/archive.go

```diff
@@ -29,6 +29,8 @@ var (
 	maxDepth   = 5
 	maxSize    = 250 * 1024 * 1024 // 20MB
 	maxTimeout = time.Duration(30) * time.Second
+
+	defaultBufferSize = 512
```

Comment:
So this is both the archive channel buffer size and the size of the elements in the channel?

Reply:
yea, at least that's how it was implemented.